### PR TITLE
[16.0][ADD] web_hide_field_with_key module

### PR DESCRIPTION
**Pull request by FranzPoize**

### CODE REVIEW

**Review by bealdav — APPROVED** [review]
Used with success since 3 months in production.
Thanks a lot for this contribution.

**Comment by vdewulf** [comment]
hello, I didn't review the code and even did'nt test this PR. I just want you to encourage to describe more in details the business context of such a module with an example. I don't get why this module could be useful for my customers.
Thanks :)

File: web_hide_field_with_key/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Hide data",
```
Comment (bealdav) [inline web_hide_field_with_key/__manifest__.py]: Wrong name here

**Comment by FranzPoize** [comment]
@vdewulf This allows to switch rapidly between confidential data and customer facing data during an exhibition for example.

**Review by PaulGoubert — APPROVED** [review]
LGTM

**Pull request merged**